src/zenml/deployers/local/local_deployer.py

```
@@ -0,0 +1,559 @@
+            port=port,
+            address=settings.address,
+            pid_file=pid_file,
+            log_file=log_file,
+        )
+
+        state = DeploymentOperationalState(
+            status=DeploymentStatus.PENDING,
+            metadata=metadata.model_dump(exclude_none=True),
+        )
+        address = settings.address
+        if address == "0.0.0.0":
```

Ruff (0.14.1)

Possible binding to all interfaces

(S104)